planet_explorer/pe_plugin.py

```diff
@@ -589,16 +589,19 @@ def show_terms(self, _):
         self._terms_browser.show()
 
     def login(self):
-        if Qgis.QGIS_VERSION_INT >= 32000 and platform.system() == "Darwin":
+        if (
+            Qgis.QGIS_VERSION_INT in [32000, 32010, 32020, 32030]
+            and platform.system() == "Darwin"
+        ):
             text = (
                 "WARNING: Your configuration may encounter serious issues with the"
-                " Planet QGIS Plugin using QGIS V3.20. We are actively troubleshooting"
-                " the issue with the QGIS team, you can track <a"
-                " href='https://github.com/qgis/QGIS/issues/44182'>Issue 44182"
-                " here</a>. In the meantime, we recommend that you use a QGIS version"
-                " between 3.10 and 3.20, such as the 3.16 long term stable release. For"
-                " further information including instructions on how to downgrade QGIS,"
-                " please refer to our <a"
+                " Planet QGIS Plugin using QGIS v3.20. These issues are documented in"
+                " <a href='https://github.com/qgis/QGIS/issues/44182'>Issue 44182"
+                " here</a>. We recommend that you use a QGIS version"
+                " other than v3.20, such as the 3.16 long term stable release, "
```

Comment:
3.22 is a long term stable release that is out, so probably better to recommend that one. I think I can test quickly if it works.